[Term Entry] Add jacobs law (#6431)
Codecademy/docs: content/uiux/concepts/ux-psychology/terms/jacobs-law/jacobs-law.md

---

> Title: "Jacob's Law"

Reviewer: Please avoid using `""` instead of `''` to keep the content consistent with other doc entries.

```suggestion
Title: 'Jacobs Law'
```

> Description: "Users prefer your site to work like others they already know, because they spend most of their time on other sites."

Reviewer: Please avoid using `""` instead of `''` to keep the content consistent with other doc entries.

```suggestion
Description: 'Jacobs Law states that users expect new interfaces to work like the ones they already know, making familiarity essential for a smooth experience.'
```

Subjects:
  - 'Computer Science'
  - 'Web Design'
Tags:
  - 'Design'
  - 'UI'
  - 'UX'
CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

> **Jacob’s Law**, proposed by Jakob Nielsen, highlights an important principle in UX Design: users spend most of their time on other websites and applications, which shapes their expectations for how things should work. Rather than creating entirely unique designs or interfaces, aligning with familiar patterns and conventions leads to smoother user experiences and less cognitive friction.

Reviewer: ```suggestion
**Jacob’s Law**, proposed by Jakob Nielsen, highlights a key UX design principle: users spend most of their time on other websites and applications, which shapes their expectations for how interfaces should work. Instead of creating entirely unique designs, aligning with familiar patterns and conventions results in a smoother user experience and reduces cognitive friction.
```

> This law underlines the importance of consistency and convention in design. Meeting users' expectations based on their existing experiences is how we reduce confusion and increase usability. Unfamiliar or overly original designs often lead to frustration, even if they are innovative or visually appealing.

Reviewer: ```suggestion
This law emphasizes the importance of consistency and convention in design. By meeting users' expectations based on their prior experiences, we minimize confusion and enhance usability. Even innovative or visually appealing designs can cause frustration if they deviate too much from established norms.
```

## Example

A common example of Jacob’s Law in practice is navigation design where most users expect the navigation bar to be at the top of a webpage, with a logo on the left and clickable links for main sections. Moving this navigation to an unexpected place, or using unfamiliar icons without labels, can negatively affect usability.

> We help users feel more comfortable and efficient when using our product. It is by designing interfaces that follow patterns users have seen on other platforms.

Reviewer: ```suggestion
Designing interfaces that follow familiar patterns helps users feel comfortable and navigate efficiently.
```

> 

Reviewer: You can try adding the following sections - 

**1. Why Jacob’s Law Matters**
- Explain its impact on user retention and engagement.
- Highlight how businesses benefit from intuitive design.

**2. Common Applications**
- Examples from popular platforms (e.g., e-commerce sites placing carts in the top-right).
- How mobile apps follow similar design principles across platforms.

**3. Exceptions & When to Innovate**
- When breaking conventions is acceptable (e.g., disruptive innovation, niche audiences).
- Balancing familiarity with creativity.

> ## Resources
> 
> - [Nielsen Norman Group – Jakob’s Law](https://www.nngroup.com/videos/jakobs-law-internet-user-experience/)
> - [UX Laws – Jacob’s Law](https://lawsofux.com/jakobs-law/)

Reviewer: Avoid adding links to other websites.

```suggestion

```

Author: Hi Pragati, Resource links were already there on Fitt's Law, so I added similarly to Jakob's Law.

Author: Hi @PragatiVerma18, I’ve included the resource links for Jacob’s Law to keep it consistent with the format used in Fitts’ Law. Please let me know if that’s alright — I’m happy to remove them if needed.
